### ISSUE TITLE
Nutil does not display error message for untiled TIFFs when only creating thumbnails

### ISSUE THREAD

**Issue by ingrre** [Neural-Systems-at-UIO/nutil#130]
I had a folder with untiled TIFFs (without being aware they were untiled) and used Nutil to get thumbnails of these files. Nutil 4.01 ran without error, but thumbs produced were only black and <40kb. See them here: Y:\HBP-Voucher_Rat-corticome\SubProject2_PosterioParietalCortex\12906\3-rotated-renamed\thumbnails-nutil4.01-untiled

Same apllies for v 4.03: Y:\HBP-Voucher_Rat-corticome\SubProject2_PosterioParietalCortex\12906\3-rotated-renamed\thumbnails 

Would be great if Nutil would warn for untiled TIFFs also when 'only creating thumbnails'

**Comment by leuat** [comment]
@ingrre I Ran a test on these files (and they are indeed tiled), but Nutil produced correct results. This might be due to the current errors we're experiencing on the network - if you copy a couple of the files to your local hard drive, are the thumbnails correctly produced?

**Comment by ingrre** [comment]
@leuat did you  mean to say 'correct' or 'incorrect' results? I moved ten images to C: and still only black thumbnails  (They are indeed tiled! Sorry about that).

**Comment by Majpuc** [comment]
We found out that those tilled tiffs had been tiled with a ZIP compression which is not compatible with Nutil. The raw tiff files should be tiled using LZW compression as for the other animals in this dataset.

**Comment by Majpuc** [comment]
@ingrre Is this issue solved then?

**Comment by Majpuc** [comment]
@ingrre Could you update this issue? Thks

**Comment by Majpuc** [comment]
@ingrre Do you have a tiled Tiff series LZW compressed I could test on?

**Comment by ingrre** [comment]
Here is a tiled tiff series LZW compressed: Y:\HBP-Voucher_Rat-corticome\SubProject3_Insula\13347-mrx\3-LZW-tiled-TIFF 

I just reexported this series previously mentioned: Y:\HBP-Voucher_Rat-corticome\SubProject2_PosterioParietalCortex\12906\2-TIFF-IR (with mirax viewer). Compression type: JPEG. 

I ran Nutil 0.404 to create thumbnails of these images. Nutil displayed no error and was been running for 1h. In the terminal however it seems that it might be something with JPEG compression? Screenshot attached. 
![Skjermbilde 2020-12-09 kl  10 04 39](https://user-images.githubusercontent.com/38322594/101608412-f71c3300-3a05-11eb-9e09-7670a931ed98.png). Nutil just finished (closed without a notion) and the result is found here: Y:\HBP-Voucher_Rat-corticome\SubProject2_PosterioParietalCortex\12906\2-TIFF-IR\thumbs\thumbnails

In mirax viewer I can only choose between JPEG and uncompressed - what do I do?
